packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -956,8 +966,19 @@ class _InterpolationSimulation extends Simulation {
 typedef _DirectionSetter = void Function(_AnimationDirection direction);
 
 class _RepeatingSimulation extends Simulation {
-  _RepeatingSimulation(double initialValue, this.min, this.max, this.reverse, Duration period, this.directionSetter)
-      : _periodInSeconds = period.inMicroseconds / Duration.microsecondsPerSecond,
+  _RepeatingSimulation(
+    double initialValue,
+    this.min,
+    this.max,
+    this.reverse,
+    Duration period,
+    this.directionSetter, [
+    this.repeatTimes,
+  ])  : assert(
```

Comment:
```suggestion
    this.directionSetter,
    this.repeatCount,
  )  : assert(
```

Small nit: since this is a private class that never omits the final argument, the brackets aren't needed here.